src/witan/models/dem/ccm/models.clj

```diff
@@ -167,14 +177,18 @@
      :witan/fn :ccm-mig/proj-dom-out-mig
      :witan/params {:start-year-avg-domout-mig 2003, :end-year-avg-domout-mig 2014}}
     {:witan/name :project-asfr
-     :witan/version "1.0.0"
+     :witan/version "1.1.0"
+     :witan/type :function
+     :witan/fn :ccm-fert/project-asfr
+     :witan/params {:fert-variant :fixed, :first-proj-year 2015, :last-proj-year 2018,
+                    :fert-scenario :principal-2012}}
+    {:witan/name :project-asmr
+     :witan/version "1.1.0"
```

Comment:
@sunnytownsend I remember @seb231 saying we weren't ready to give users the choice - are you sure we want to use variant-based functions for the `project-*` collection from now on?